backend/servers/express-webrouter/config/__teleport.json

```diff
@@ -0,0 +1,3 @@
+{
+  "templates": $[project.templates]
+}
```

Comment:
so now it is <REPLACE>teleport_welcome.json and it is used here https://github.com/snipsco/teleport-express-webrouter/blob/server-in-dev/backend/servers/express-webrouter/app/templates/_body.html#L55